conf/application.conf

```
@@ -1,68 +1,49 @@
-application {
-  name = webKnossos
-  branch = local
-  version = "1.0.0-1"
+http {
+  uri = "http://localhost:9000"
+  address = "127.0.0.1"
+  port = 9000
 }
 
-play.modules.enabled += "WebKnossosModule"
-play.modules.enabled += "SilhouetteModule"
-play.modules.enabled += "com.scalableminds.webknossos.datastore.DataStoreModule"
-play.modules.enabled += "com.scalableminds.webknossos.tracingstore.TracingStoreModule"
-
-datastore {
-  enabled = true
-  key = "something-secure"
-  name = "localhost"
-  publicUri = "http://localhost:9000"
-  debugMode = true
-  webKnossos {
-    uri = "http://localhost:9000"
-    pingIntervalMinutes = 10
+play {
+  modules {
+    enabled += "WebKnossosModule"
+    enabled += "SilhouetteModule"
+    enabled += "com.scalableminds.webknossos.datastore.DataStoreModule"
+    enabled += "com.scalableminds.webknossos.tracingstore.TracingStoreModule"
   }
-  fossildb {
-    address = "localhost"
-    port = 7155
+  http {
+    secret.key = "$JUeGY5yhw~pw%`pyVeDF*}`sF^kL'@sF$=dSTx?<z*f/vFg"
+    # increase resources
+    parser.maxMemoryBuffer = 250M
+    parser.maxDiskBuffer = 1G
+    filters = "com.scalableminds.webknossos.datastore.Filters"
   }
-}
 
-tracingstore {
-  enabled = true
-  key = "something-secure"
-  name = "localhost"
-  publicUri = "http://localhost:9000"
-  webKnossos {
-    uri = "http://localhost:9000"
-    pingIntervalMinutes = 10
-  }
-  fossildb {
-    address = "localhost"
-    port = 7155
-  }
-  redis {
-    address = "localhost"
-    port = 6379
-  }
-}
+  # Unset some of the security filters enabled in datastore.Filters
+  filters.headers.contentSecurityPolicy = null
+  filters.headers.contentTypeOptions = null
+  filters.headers.frameOptions = null
```

It's not clear to me yet which parts should be grouped and which not. E.g. why not the following?
```suggestion
  filters.headers {
    # Unset some of the security filters enabled in datastore.Filters
    contentSecurityPolicy = null
    contentTypeOptions = null
    frameOptions = null
  }
```

---

Good point! Let’s group all the things (that have more than one item)